prov/efa/src/efa_ep.c

```diff
@@ -227,12 +227,6 @@ static int efa_ep_bind(struct fid *fid, struct fid *bfid, uint64_t flags)
 
 	switch (bfid->fclass) {
 	case FI_CLASS_CQ:
-		if (flags & FI_SELECTIVE_COMPLETION) {
```

Comment:
Why did we have this in the first place? Because dgram doesn't support FI_SELECTIVE_COMPLETION?

Reply:
Yes

Reply:
And the efa_ep PR was merged before the FI_CONTEXT2 change, so I didn't remove it.